src/app/todolist/todolist.component.ts

```diff
@@ -11,18 +11,18 @@ import { LoggedinService } from '../services/loggedin.service'
 })
 
 export class TodolistComponent implements OnInit {
-  todos: TodoForm[] = []
+  todolist?: TodoForm[] = []
   loggedUser = this.userName.getLogged()
   constructor(private todoService: TodoService, public userName: LoggedinService) { }
 
   ngOnInit(): void {
-    this.todoService.getTodoList().subscribe(todolist => { this.todos = todolist.todos; console.log(this.todos), console.log(todolist) })
+    this.todoService.getTodoList().subscribe(todolist => { this.todolist = todolist.body?.todos; console.log(this.todolist), console.log(todolist) })
```

Comment:
no need to keep the logs here if you've finished debugging
usually no console.log code should be pushed into master